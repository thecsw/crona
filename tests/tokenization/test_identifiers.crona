while1
_thisIsVal1d
3thisIs_Not
This_1_isThough