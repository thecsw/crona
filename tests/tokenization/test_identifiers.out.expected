ID:while1 [1,1]
ID:_thisIsVal1d [2,1]
INTLIT:3 [3,1]
ID:thisIs_Not [3,2]
ID:This_1_isThough [4,1]
EOF [4,16]
